equistore-core/Cargo.toml

```diff
@@ -11,7 +11,8 @@ exclude = [
 [lib]
 # When our minimal Rust version becomes 1.64, we can pass these options directly
 # to Cargo. Until then, we build all the crate-type we need.
-crate-type = ["cdylib", "staticlib", "rlib"]
+name = "equistore"
```

Comment:
`equistore` or `equistore-core`. Maybe the first becuae there is no `equistore-core` is the only rust package, no?

Reply:
so that's the main change here: by setting the name of the lib to `equistore`, rustc will create equistore.dll and equistore.dll.lib directly. Before it was using the default `equistore_core` as the name, and we had to rename the files